### PR TITLE
Document supported platforms and status of project

### LINKED ISSUE #149: document package maturity
This project seems very cool!

However it's hard to assess the relative maturity of this package; which features it has or doesn't have, what are the largest caveats or roadblocks that one might encounter. 

It would be super great to have some high-level documentation about what consumers of this package can expect so that people can assess how suitable/viable this package is. 

Thanks! :)

### PR DESCRIPTION
**Pull request by TotallyGamerJet**
Closes #149

### CODE REVIEW

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

File: README.md [hunk 1]
```diff
@@ -17,6 +21,13 @@ born to bring that same vision to the other platforms supported by Ebitengine.
 - **Dynamic Linking**: Load symbols at runtime and use it as a plugin system.
 - **Foreign Function Interface**: Call into other languages that are compiled into shared objects.
 
+## Supported Platforms
+
+- **Darwin**: amd64, arm64
```
Comment (hajimehoshi) [inline README.md:26]: Maybe macOS is better? (If we want to use GOOS names, let's unify them with lower cases)

File: README.md [hunk 2]
```diff
@@ -17,6 +21,15 @@ born to bring that same vision to the other platforms supported by Ebitengine.
 - **Dynamic Linking**: Load symbols at runtime and use it as a plugin system.
 - **Foreign Function Interface**: Call into other languages that are compiled into shared objects.
 
+## Supported Platforms
+
+- **FreeBSD**: amd64, arm64
+- **Linux**: amd64, arm64
+- **macOS**: amd64, arm64
```
Comment (hajimehoshi) [inline README.md:28]: Sorry I forgot about iOS.

`macOS/iOS`?

**Review by hajimehoshi — COMMENTED** [review]
Offtopic: has anyone tested purego with Android?

File: README.md (hunk 2 above)
Comment (TotallyGamerJet) [reply README.md:28]: oh true.... How did I forget that!

**Comment by TotallyGamerJet** [comment]
> Offtopic: has anyone tested purego with Android?

Does the Android version of Ebitengine use any purego?

**Comment by hajimehoshi** [comment]
> Does the Android version of Ebitengine use any purego?

Not yet. Calling JNI without Cgo might be pretty nice.

**Comment by TotallyGamerJet** [comment]
> Not yet. Calling JNI without Cgo might be pretty nice.

But would it suffer the same issue as iOS where you kind of have to use Cgo?

**Comment by hajimehoshi** [comment]
> But would it suffer the same issue as iOS where you kind of have to use Cgo?

Yes Cgo is inevitable, but we can reduce them, like what we did for Oto.

**Review by hajimehoshi — APPROVED** [review]
LGTM!

**Pull request merged**